pkg/config/config.go

```diff
@@ -353,6 +353,18 @@ func InitConfig(config Config) {
 		} else {
 			config.SetDefault("container_cgroup_root", "/sys/fs/cgroup/")
 		}
+
+		if pathExists("/host/etc") {
+			if val, isSet := os.LookupEnv("HOST_ETC"); !isSet {
+				// We want to detech the host kernel instead of the one from the container. 'HOST_ETC' is Used
+				// by some librairies like gopsutil and by the system-probe to download the right kernel
+				// headers.
```

Comment:
```suggestion
				// We want to detect the host kernel instead of the one from the container. 'HOST_ETC' is used
				// by some libraries like gopsutil and by the system-probe to download the right kernel
				// headers.
```